ui/src/app/components/prepare-migration/prepare-migration.component.html

```diff
@@ -51,7 +61,6 @@ <h2 *ngIf="isSharded">Source and Target Database definitions (per shard)</h2>
             </mat-option>
         </mat-select>
     </mat-form-field>
-    <br />
```

Comment:
Why is this removed?

Reply:
> Why is this removed?

I removed the br tag in line 45 due to the same reasons that I stated in the previous reply and this tag was just the closing of it. However it won't have affected the code, doing this seemed more cleaner.